src/components/routes/Member/AuthenticatedMember/UpdateCredentialsRoute/UpdateCredentialsContent/UpdateUsernameForm/UpdateUsernameForm.jsx

```diff
@@ -55,7 +55,7 @@ const UpdateUsernameForm = (props) => {
 
   const renderSubmitButton = () => (
     <div className="update-password__button-container">
-      <Button className="button__secondary-color update-password_button" type="submit" value="Update" />
+      <Button className="button button button__small button__blue" type="submit" value="Update" />
```

Comment:
why the same button class used twice?
```suggestion
      <Button className="button button__small button__blue" type="submit" value="Update" />
```

Reply:
done